cinema/models.py

```diff
@@ -35,12 +39,21 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, ext = os.path.splitext(filename)
+
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{ext}"
+
+    return os.path.join("uploads/movies/", filename)
```

Comment:
Do this code crossplatform
```suggestion
    return os.path.join("uploads", "movies", filename)
```